components/disclosure/circle-detail-tabs.tsx

```diff
@@ -137,13 +191,81 @@ export const CircleDetailTabs: FC<CircleDetailTabsProps> = ({
           </Indicator>
         </Tab>
       </TabList>
-
       <TabPanels>
         <TabPanel>
           <CircleActivitydays />
         </TabPanel>
         <TabPanel>画像</TabPanel>
-        <TabPanel>掲示板</TabPanel>
+
+        <TabPanel>
+          <MultiSelect
+            placeholder="項目を選択"
+            component={({ label, onRemove }) => (
+              <Tag onClose={onRemove}>{label}</Tag>
+            )}
+            onChange={setSelectedOptions}
+            value={selectedOptions}
+          >
+            <Option value="スレッド">スレッド</Option>
+            <Option value="お知らせ">お知らせ</Option>
+            <Option value="重要">重要</Option>
+          </MultiSelect>
+          <SimpleGrid w="full" columns={1} gap="md">
+            {dummyData
+              .filter((item) => {
+                if (selectedOptions.length === 0) {
+                  return true
+                }
+                // "重要"のみ
+                if (
+                  selectedOptions.includes("重要") &&
+                  selectedOptions.length === 1
+                ) {
+                  return item.isImportant
+                }
+                // "重要"と"スレッド"
+                if (
+                  selectedOptions.includes("重要") &&
+                  selectedOptions.includes("スレッド")
+                ) {
+                  return item.type === "スレッド" && item.isImportant
+                }
+                // "重要"と"お知らせ"
+                if (
+                  selectedOptions.includes("重要") &&
+                  selectedOptions.includes("お知らせ")
+                ) {
+                  return item.type === "お知らせ" && item.isImportant
+                }
+                // "スレッド"or"お知らせ"
+                return selectedOptions.includes(item.type)
+              })
+              .map((item) => (
+                <GridItem key={item.id} w="full" rounded="md" as={Card}>
+                  <CardBody position="relative">
+                    <HStack gap="4" align="center">
+                      <Avatar src={item.icon} />
+                      {item.isImportant && (
+                        <Badge colorScheme="red">重要</Badge>
+                      )}
+                      <Text fontWeight="bold">{item.title}</Text>
+                    </HStack>
+                    <VStack
+                      position="absolute"
+                      bottom="4px"
+                      right="4px"
+                      align="end"
+                    >
+                      <Text fontSize="sm" color="gray.500">
+                        {item.date} {item.time}
+                      </Text>
+                    </VStack>
```

Comment:
`position: relative;`・``position: absolute;`で無理やり固定しないでください。

```suggestion
                  <CardBody>
                    <HStack w="full" gap="4" justifyContent="space-around">
                      <HStack w="full">
                        <Avatar src={item.icon} />
                        {item.isImportant && (
                          <Badge colorScheme="red">重要</Badge>
                        )}
                        <Text fontWeight="bold">{item.title}</Text>
                      </HStack>
                      <VStack
                        align="end"
                      >
                        <Text fontSize="sm" color="gray.500">
                          {item.date} {item.time}
                        </Text>
                      </VStack>
                    </HStack>
```